docs/openapi.yaml

```diff
@@ -584,7 +584,113 @@ paths:
         responses: { }
         httpMethod: POST
         type: AWS_PROXY
-
+  /{publicationIdentifier}/filelink/{fileIdentifier}:
```

Comment:
because you get a object with file "links", not a file. I would like to redo this in the future™️ so that we actually returns the file here and then it's also convenient that /file is available. I would like to return file directly here so that google scholar have a chance so index our file content